js/src/util.js

```diff
@@ -50,11 +45,9 @@ const Util = (($) => {
 
     const el = document.createElement('bootstrap')
```

Comment:
they are no needs now to create an element here just return : 
```js
return {
  end: 'transitionend'
}
```

that's all

Reply:
@Johann-S done and committed.